pipelines/pom.xml

```diff
@@ -375,6 +375,10 @@
           <compilerArgs>
             <!-- Enable access to the packages at compile time -->
             <arg>--add-exports=java.base/jdk.internal.misc=ALL-UNNAMED</arg>
+            <!-- To retain parameter names for compiled classes, this is needed for latest spring.
+            See https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-6.x#parameter-name-retention
+            for the details -->
+            <arg>-parameters</arg>
```

Comment:
Question: Why are you not using the `<parameters>true</parameters>` variant which is suggested in Spring's [documentation](https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-6.x#parameter-name-retention) instead?

Reply:
I didn't have specific reasons, just that in some of the examples I had referred like [this](https://github.com/spring-projects/spring-boot/issues/9323) had this usage. Made the changes as suggested in Spring's [documentation](https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-6.x#parameter-name-retention) now.